acceptance/helpers/FirefoxDriver.js

```diff
@@ -50,8 +50,16 @@ class FirefoxDriver {
         }
     }
 
+    getUsername() {
+        return this.username;
+    }
+
+    async getDriver() {
+        await this.driver;
+    }
```

Comment:
If this isn't being used can we get rid?
```suggestion
```

Reply:
I don't have a strong opinion, can we get it merged?

Reply:
I'll drop that out and pull in development 👍